packages/module/src/components/AccessConsoles/AccessConsoles.tsx

```diff
@@ -21,6 +19,22 @@ const isChildOfType = (child: any, type: string) => {
   return false;
 };
 
+const useStyles = createUseStyles({
+  console: {
+    display: 'grid',
+    gridTemplateAreas:
+      `'actions-main actions-extra' + 'main main'`,
+      rowGap: 'var(--pf-v6-global--spacer--md)'
```

Comment:
These global variables will not exist in v6 - they are replaced with things that start with `--pf-t-`

Reply:
done.